about-me.html

```diff
@@ -0,0 +1,56 @@
+<html>
+<head>
+	<title>Saskia West Software Dev Portfolio</title>
+	<link href="normalize.css" rel="stylesheet" type="text/css">
+	<link href="style.css" rel="stylesheet" type="text/css">
+	<link href="https://fonts.googleapis.com/css2?family=Cinzel&family=Girassol&family=Gloria+Hallelujah&family=Montserrat:wght@200&family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+	
+<section class="top-bar">
+	
+	<nav class="sw-logo">S W</nav>
+	
+	<nav class="nav-bar">
+		<a href="index.html">Home</a>
+		<a href="about-me.html">About Me</a>
+		<a href="portfolio.html">Portfolio</a>
+		<a href="contact.html">Contact</a>
+	</nav>
+	
+	<button class="hamburger">Menu</button>
+	
+</section>
+	
+<main class="about-main">
```

Comment:
you will only have one main tag, so you can only target it straight away, maybe you don't need the class then?